evalml/pipelines/components/transformers/preprocessing/time_series_featurizer.py

```diff
@@ -215,6 +218,22 @@ def _compute_rolling_transforms(self, X, y, original_features):
         )
         return data
 
+    def _delay_df(
+        self,
+        data,
+        cols_to_delay,
+        categorical_columns=None,
+        X_categorical=None,
+    ):
+        lagged_features = {}
+        for col_name in cols_to_delay:
+            col = data[col_name]
+            if categorical_columns and col_name in categorical_columns:
+                col = X_categorical[col_name]
+            for t in self.statistically_significant_lags:
+                lagged_features[f"{col_name}_delay_{t}"] = col.shift(t)
```

Comment:
We're not going to be doing any external matching on this name format, right?  If so, I think we might want to establish a pattern of making this string format like a module level thing or accessible via the class

Reply:
Good call - adjusting!